src/hooks/emailNotification/useEmailNotification.js

```diff
@@ -0,0 +1,18 @@
+import { useSelector } from 'react-redux';
+
+/**
+ * useEmailNotification hook
+ *
+ * This hook returns the current status of email Notification that is stored in the Redux store in the
```

Comment:
@Tishasoumya-02 Also please fix the grammer! :)

Reply:
yup my mistake